<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>231</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="setprocess_num">
       <property name="text">
        <string>设置进程数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinRowcount">
       <property name="font">
        <font>
         <kerning>true</kerning>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_process">
       <property name="text">
        <string>加一个进程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sub_process">
       <property name="text">
        <string>去掉一个进程</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="read_excel">
       <property name="text">
        <string>表格初始化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clear_num">
       <property name="text">
        <string>清除数据</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableInfo">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>481</width>
      <height>201</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>481</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 290 9pt &quot;Microsoft YaHei UI&quot;;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>827</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
